index.html

```diff
@@ -11,37 +11,203 @@
+        <h1 class="heading_title">My Logo</h1>
         <nav class="navbar">
             <img class="img2" src="resources/menu.png" alt="Menu Icon">
         </nav>
     </header>
-    <section>
-        <div class="description_section">
-            <h1 class="h1_heading">Hey There.<br>
-                I’m Alex.</h1>
-            <h3 class="h3_heading">I'm a Software Developer</h3>
-            <p class="paragraph">I can help you build a product , feature or website Look through some of my work and
-                experience! If you like what you see and have a project you need coded,don’t hesitate to contact me.</p>
-            <div>
-                <ul class="social">
-                    <li class="lists"><a href="#"  > <img src="./resources/logo1.png" alt="logo"> </a></li>
-                    <li class="lists"><a href="#"  > <img src="./resources/logo2.png" alt="logo"> </a></li>
-                    <li class="lists"><a href="#" > <img src="./resources/logo3.png" alt="logo"> </a></li>
-                    <li class="lists"><a href="#"  > <img src="./resources/logo4.png" alt="logo"> </a>></li>
-                    <li class="lists"><a href="#"  > <img src="./resources/logo5.png" alt="logo"> </a></li>
-                </ul>
-            </div>
-            <div class="image_container">
-                <img src="resources/Illustration.png" alt="image">
+
+    <!-- the main section starts from here -->
+    <main>
+        <section>
+            <div class="description_section">
+                <h1 class="h1_heading">Hey There.<br>
+                    I’m Alex.</h1>
+                <h3 class="h3_heading">I'm a Software Developer</h3>
+                <p class="paragraph">I can help you build a product , feature or website Look through some of my work
+                    and
+                    experience! If you like what you see and have a project you need coded,don’t hesitate to contact me.
+                </p>
+                <div>
+                    <ul class="social">
+                        <li class="lists"><a href="#"> <img src="./resources/logo1.png" alt="logo"> </a></li>
+                        <li class="lists"><a href="#"> <img src="./resources/logo2.png" alt="logo"> </a></li>
+                        <li class="lists"><a href="#"> <img src="./resources/logo3.png" alt="logo"> </a></li>
+                        <li class="lists"><a href="#"> <img src="./resources/logo4.png" alt="logo"> </a>></li>
+                        <li class="lists"><a href="#"> <img src="./resources/logo5.png" alt="logo"> </a></li>
+                    </ul>
+                </div>
+                <div class="image_container">
+                    <img src="resources/Illustration.png" alt="image">
+                </div>
             </div>
+        </section>
+        <div class="Second_part">
+            <section class="work_section">
+                <div class="work_section_title">My Recent Works </div>
```

Comment:
- [ ] Kindly I recommend you to  uses a header tag `hx(h2)` tag for the title (e.g., "My Recent Works") to follow the Best Practice of HTML and CSS.

Reply:
@Abenezer-Tilahun Done!